### PR TITLE
docs: fourth lol ppp

### LINKED ISSUE #144: Create fourth ppp

### PR DESCRIPTION
**Pull request by holzeis**
resolves #144

### CODE REVIEW

File: docs/ppp/iteration-4-08112022.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+### Plans 📆
+
+- Fund the on-chain wallet - https://github.com/itchysats/10101/issues/113
+- Setup and build the maker lightning node boilerplate - https://github.com/itchysats/10101/issues/89
+- Open a channel between taker and maker - https://github.com/itchysats/10101/issues/10
+- Persisting custom outputs - https://github.com/itchysats/rust-lightning/issues/2
+
```
Comment (klochowicz) [inline docs/ppp/iteration-4-08112022.md:7]: quite important plan missing: get the rust-lightning working without our 10101 app

File: docs/ppp/iteration-4-08112022.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+### Progress ✅
+
+- Wired the backup seed screens with the rust library managing the wallet. - https://github.com/itchysats/10101/pull/102
+- Finished the trade view flow with mock data. - [Create CFD Offer Screen](https://github.com/itchysats/10101/issues/87), [Create CFD Overview Screen](https://github.com/itchysats/10101/issues/115)
+- Published our first release [TODO: add link to github packages here](https://addlink.github.com) for android.
+- Prototypically added custom output to a lightning channel show casing how a DLC extension could work.
+- Designed a funding wallet wire frame workflow [TODO: add screenshots here?]
+- Built a wallet library combining bdk and ldk. https://github.com/itchysats/bdk-ldk/pull/1
```
Comment (klochowicz) [inline docs/ppp/iteration-4-08112022.md:15]: we didn't built it, we only upgraded it to the latest version and reworked it.

**Review by klochowicz — COMMENTED** [review]
thanks for writing this up, I've added a few comments

File: docs/ppp/iteration-4-08112022.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+### Plans 📆
+
+- Fund the on-chain wallet - https://github.com/itchysats/10101/issues/113
+- Setup and build the maker lightning node boilerplate - https://github.com/itchysats/10101/issues/89
+- Open a channel between taker and maker - https://github.com/itchysats/10101/issues/10
+- Persisting custom outputs - https://github.com/itchysats/rust-lightning/issues/2
```
[the remark this answers is not in the document]
Comment (luckysori) [reply docs/ppp/iteration-4-08112022.md:6]:
```suggestion
- Creating a DLC using our newly introduced `add_custom_output` protocol - https://github.com/itchysats/rust-lightning/issues/2
```

**Comment by da-kami** [comment]
Taking this over, finishing and then posting it :)
Update with image coming shortly.

**Pull request merged**